### DISCUSSION TITLE
SOFAServerless是怎样支持Tomcat的？

### DISCUSSION THREAD

**Discussion by Aurora178** [sofastack/sofa-serverless#459] [Q&A] [answered]
主要是想知道SOFAServerless支持Tomcat的具体形式是啥？只是部署应用在tomcat实例上吗？SOFAServerless是如何支持Tomcat的？

**Comment by lvjing2** [comment, marked as answer]
你好，可以查看这里
https://github.com/sofastack/sofa-serverless/blob/master/samples/springboot-samples/web/tomcat/README.md

**Reply by Aurora178** [reply]
你好，这块内容我看过了。我有个问题？就是如果换成payara这类中间件的话，类似吗？

**Reply by Aurora178** [reply]
这里的支持tomcat也是指内置的tomcat吧？

**Reply by lvjing2** [reply]
也是类似的，不过也是有个适配的逻辑。这个 smaples 里的tomcat是 springboot 内置的

**Reply by Aurora178** [reply]
好的，谢谢
